app/assets/stylesheets/custom.scss

```
@@ -71,3 +71,50 @@ p {
   padding-top: 9px;
   font-weight: bold;
 }
+
+/* microposts */
+.microposts-list {
```

細かいけど、`{単数形}-list` かなと。
```suggestion
.micropost-list {
```